go/cli/mcap/cmd/filter.go

```diff
@@ -406,6 +469,12 @@ usage:
 			[]string{},
 			"messages with topic names matching this regex will be excluded, can be supplied multiple times",
 		)
+		includeLastPerChannelTopics := filterCmd.PersistentFlags().StringArray(
+			"include-last-per-channel-topic-regex",
+			[]string{},
+			"For included topics matching this regex, the most recent message previous to the start time"+
```

Comment:
nit: prior sounds more natural to me
```suggestion
			"For included topics matching this regex, the most recent message prior to the start time"+
```